app/graphql/types/inputs/pool_samples_input_type.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Types
+  module Inputs
+    # The input arguments required to pool samples.
+    class PoolSamplesInputType < BaseInputObject
+      argument :plate_id, Int, required: true
+      argument :tag_set, Int, required: true # 24 or 96
```

Comment:
If we make both of these be required, you won't be able to use this input type to do "update" migrations in the future unless the user always updates both the plate ID and the tag set.  Maybe that's OK?  I avoided specifying that input fields be required so that updates could update just one value or the other.